test/Microsoft.TemplateEngine.Cli.UnitTests/TemplateResolutionTests/MultiShortNameResolutionTests.cs

```diff
@@ -25,8 +26,9 @@ public void AllTemplatesInGroupUseAllShortNamesForResolution()
                 INewCommandInput userInputs = new MockNewCommandInput(testShortName);
 
                 TemplateResolutionResult matchResult = TemplateResolver.GetTemplateResolutionResult(MultiShortNameGroupTemplateInfo, new MockHostSpecificDataLoader(), userInputs, "C#");
-                matchResult.TryGetCoreMatchedTemplatesWithDisposition(x => x.IsMatch, out IReadOnlyList<ITemplateMatchInfo> matchedTemplateList);
+                matchResult.TryGetUnambiguousTemplateGroupToUse(out IReadOnlyList<ITemplateMatchInfo> matchedTemplateList);
```

Comment:
Should the boolean outcome of TryGetUnambiguousTemplateGroupToUse also be asserted?